lib/jekyll/drops/drop.rb

```diff
@@ -211,6 +212,10 @@ def fetch(key, default = nil, &block)
         return yield(key) unless block.nil?
         return default unless default.nil?
       end
+
+      def fallback_data
+        @fallback_data ||= {}
+      end
```

Comment:
The base drop class assumes fallback data to exist and return a hash. Stubbed out the default value so that child classes don't each have to implement it if they don't have fallback data (including our test fixture).